Composer/packages/client/src/pages/publish/PublishStatusList.tsx

```diff
@@ -54,16 +47,12 @@ function onRenderDetailsHeader(props, defaultRender) {
 }
 
 export const PublishStatusList: React.FC<IStatusListProps> = (props) => {
-  const { items, onItemClick, groups } = props;
-  const [selectIndex, setSelectedIndex] = useState<number>();
+  const { items, onLogClick, onRollbackClick } = props;
   const [currentSort, setSort] = useState({ key: 'PublishDate', descending: true });
   const sortByDate = (ev: React.MouseEvent<HTMLElement>, column: IColumn): void => {
```

Comment:
https://codepen.io/pen/ . Why do we need to write custom functionality for sorting by date? If u look at this pen its built in DetailsList

Reply:
I have checked the sort function. If we dont give custom functionality, the list can not sort by default. 
![detailsort](https://user-images.githubusercontent.com/5860999/100057461-73d5db80-2e62-11eb-9668-6165254b7b15.gif)